### DISCUSSION TITLE
Does Quasar FRamework support  webrtc

### DISCUSSION THREAD

**Discussion by mmkecibi** [quasarframework/quasar#11138] [CLI - Capacitor / Cordova modes]
Does Quasar FRamework support  webrtc? 
Is there any  webrtc. cordova plugin  for quasat ios and android?

**Comment by Dirk-** [comment]
You can build Vue components with WebRTC features ([here is an example](https://cloudinary.com/blog/building_a_camera_app_with_webrtc_vue_js_and_cloudinary)) and then just use them in a Quasar app.
